test/gtest/common/test.h

```diff
@@ -287,11 +287,8 @@ class GTEST_TEST_CLASS_NAME_(test_case_name, test_name) : public test_case_name
   static ::testing::TestInfo* const test_info_;\
   GTEST_DISALLOW_COPY_AND_ASSIGN_(\
       GTEST_TEST_CLASS_NAME_(test_case_name, test_name));\
-}; \
-\
-::testing::TestInfo* const GTEST_TEST_CLASS_NAME_(test_case_name, test_name)\
-  ::test_info_ = \
-    ::testing::internal::MakeAndRegisterTestInfo( \
+  static ::testing::TestInfo *__attribute__((noinline)) create_test_info() { \
```

Comment:
can we use GTEST_NO_INLINE_ to replace `__attribute__((noinline))`?

Reply:
done